modules/openapi-generator/src/test/java/org/openapitools/codegen/java/apachehttpclient/ApacheHttpClientCodegenTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
This line is not needed per https://opensource.stackexchange.com/a/7301/12997. Will remove it in a separate PR.

Reply:
UPDATE: filed https://github.com/OpenAPITools/openapi-generator/pull/14098